test-configs.yaml

```diff
@@ -1415,6 +1440,14 @@ test_configs:
       - boot
       - simple
 
+  - device_type: meson-g12b-a311d-khadas-vim3
```

Comment:
This board doesn't appear in the staging.kernelci.org results: https://staging.kernelci.org/boot/all/job/kernelci/branch/staging.kernelci.org/kernel/staging-20191106.0/
Is it online?

Reply:
A coworker left it in maintenance. It is now up in good state.

Reply:
OK thanks, it should show up in the next round of tests then.

Reply:
Tested OK on staging: https://staging.kernelci.org/boot/id/5dc5603fcf3a0f7afe96686b/